layers/+tools/finance/packages.el

```diff
@@ -60,4 +60,5 @@
       ;; more info: https://github.com/emacs-evil/evil/issues/301
       ;; TODO remove this hack if the limitation is removed upstream
       (add-hook 'ledger-mode-hook 'evil-normalize-keymaps)
+      (setq pcomplete-termination-string "")
```

Comment:
You are changing the global value here. It should instead be changed locally in a hook.

Reply:
@Alexander-Miller Thanks for review. Ill incorporate the changes pointed out.

Reply:
@Alexander-Miller  Done